companion-app/src/Components/UploadFile.js

```diff
@@ -16,9 +14,22 @@ let packet = {
   ],
   filename: "",
   play: false,
-//  stop: false,
+  stop: false,
   parameters: {
-    volume: 0.0
+    volume: 0.0,
+    echo: {
+      apply: false,
+      delay: 10.0,
+      feedback: 0.0,
+      dry: 0.0,
+      wet: 0.0
+    },
+    equalizer: {
+      apply: false,
+      lowgain: 0.0,
+      midgain: 0.0,
+      highgain: 0.0
+    }
   }
 };
```

Comment:
This seems to match the format we are going for when it comes to audio playback. We just need to ensure that this JSON format can be parsed with the PacketParser class that Brandon wrote.